src/SchemaForm.js

```diff
@@ -58,21 +66,21 @@ class SchemaForm extends React.Component {
             return null;
         }
 
-        const key = form.key && form.key.join(".") || index;
+        const key = form.key && form.key.join('.') || index;
 
-        const {errors} = this.props;
+        const errors = this.props && this.props.errors ? this.props.errors : {};
```

Comment:
when would we not have `this.props`?  Could this be just `const errors = this.props.errors || {}`?

Reply:
this is the problem that components sometimes are not having props. it's some kind of temporary state of time and not breaking anything, but such case are exist and I still don't know why it's there